Microsoft.Toolkit.Uwp.UI.Controls/RotatorTile/RotatorTile.cs

```diff
@@ -153,7 +165,6 @@ private void RotatorTile_Unloaded(object sender, RoutedEventArgs e)
         private void Timer_Tick(object sender, object e)
         {
             var item = GetItemAt(_currentIndex + 1);
-            _timer.Interval = TimeSpan.FromSeconds(_randomizer.Next(5) + 5); // randomize next flip
```

Comment:
Interval should be updated on each tick, or the randomization isn't changing. ie
```
_timer.Interval = GetTileDuration();
```